.github/ISSSUE_TEMPLATE/99_other.md

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
I don't think we need this template; unless you disable it, people will see an 'open generic issue' at the bottom of the list of templates.

![Uploading Screenshot 2023-02-08 at 12.45.55 PM.png…]()